### PR TITLE
Update hello world example and remove other outdated GPT examples

### PR DESCRIPTION
**Pull request by mkendall07**
## Type of change

- [x] Refactoring (no functional changes, no api changes)

## Description of change
This cleans up the GPT example pages by removing the old pattern of polling to see if GPT is ready and also removes bidder specific pages because it's not realistic to keep them all up date. Per Google, this is no longer necessary (https://developers.google.com/doubleclick-gpt/versions): 

> As of GPT v2019032001, Pubads service is now fully operational immediately after calling googletag.enableServices() instead of being initialized asynchronously. This means that googletag.pubadsReady is now guaranteed to be true right after calling googletag.enableServices(). Polling to check the value of googletag.pubadsReady should no longer be necessary.

### CODE REVIEW

**Comment by mkendall07** [comment]
This pull request **fixes 10 alerts** when merging 84b042486f54b9d3c1fcd6b411472fff591588d8 into d3643fd4a1a5d89743751a2c3c5909ef365e09ce - [view on LGTM.com](https://lgtm.com/projects/g/prebid/Prebid.js/rev/pr-c7fab28d380742c889b45402c65117406f6dc7ed)

**fixed alerts:**

* 10 for Unused variable, import, function or class

---

This project has automated code review enabled, but doesn't use the <a href="https://github.com/apps/lgtm-com">LGTM GitHub App</a>. Migrate over by <a href="https://github.com/apps/lgtm-com/installations/new">installing the app</a>. Read about the <a href="https://lgtm.com/blog/announcing_github_apps_migration">benefits of migrating to GitHub Apps</a> in the blog.

---

*Comment posted by [LGTM.com](https://lgtm.com)*

File: integrationExamples/gpt/hello_world.html [hunk 1]
```diff
@@ -9,8 +9,10 @@
 
 <html>
   <head>
+      <script async src="../../build/dev/prebid.js"></script>
+      <script async src="https://www.googletagservices.com/tag/js/gpt.js"></script>
       <script>
-        var PREBID_TIMEOUT = 3300;
+        var FAILSAFE_TIMEOUT = 3300;
```
Comment (robertrmartinez) [inline integrationExamples/gpt/hello_world.html]: +1 on renaming this, had heard people confused thinking it was bidder timeout and others!

**Comment by bretg** [comment]
@mkendall07 - I made a couple of updates
- specifically passing the PREBID_TIMEOUT into requestBids()
- added in the FAILSAFE settimeout on the GDPR example... why was it commented out?

**Comment by mkendall07** [comment]
This pull request **fixes 10 alerts** when merging 84047d6d312dffc54e8b827cdeeee9c015aa261f into e6e080a7097677288a472a05fcfbb92da8242aeb - [view on LGTM.com](https://lgtm.com/projects/g/prebid/Prebid.js/rev/pr-63a842a38b09b29b3209668690abcc64764b8df8)

**fixed alerts:**

* 10 for Unused variable, import, function or class

---

This project has automated code review enabled, but doesn't use the <a href="https://github.com/apps/lgtm-com">LGTM GitHub App</a>. Migrate over by <a href="https://github.com/apps/lgtm-com/installations/new">installing the app</a>. Read about the <a href="https://lgtm.com/blog/announcing_github_apps_migration">benefits of migrating to GitHub Apps</a> in the blog.

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by bretg** [comment]
Back to you @mkendall07 to confirm the timeout changes I made.

**Comment by mkendall07** [comment]
LGTM

**Pull request merged**